src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.sh

```diff
@@ -34,9 +34,9 @@ __PORTABLETARGETS_PROJECT_CONTENT="
 
 __PUBLISH_TFM=netcoreapp2.0
 
-__DEFAULT_RESTORE_ARGS="--no-cache --packages \"${__PACKAGES_DIR}\""
-__INIT_TOOLS_RESTORE_ARGS="${__DEFAULT_RESTORE_ARGS} --source https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json --source https://api.nuget.org/v3/index.json ${__INIT_TOOLS_RESTORE_ARGS:-}"
-__TOOLRUNTIME_RESTORE_ARGS="--source https://dotnet.myget.org/F/dotnet-core/api/v3/index.json ${__INIT_TOOLS_RESTORE_ARGS}"
+__DEFAULT_RESTORE_ARGS="--no-cache --packages"
```

Comment:
Do we really need to remove __PACKAGES_DIR from this variable?

Reply:
It was required for the path with spaces subcase. if we don't supply the path variable directly, it was being treated as separate argument, the quotes was getting removed by the eval command and dotnet was treating it as a new argument

Reply:
Maybe then let's leave `--packages` out of this variable. As it makes no sense to use `__DEFAULT_RESTORE_ARGS` by itself. You always have to append the __PACKAGES_DIR after using it.

So remove `--packages` here, and put it into all the places using this variable.
ex.

```
${__DEFAULT_RESTORE_ARGS} --packages \"${__PACKAGES_DIR}\"
```